Resources/Prototypes/Maps/syndetroit.yml

```diff
@@ -0,0 +1,54 @@
+- type: gameMap
+  id: Syndetroit
+  mapName: 'Syndetroit'
+  mapPath: /Maps/syndetroit.yml
+  minPlayers: 30
+  maxPlayers: 60
+  stations:
+    Barratry:
```

Comment:
Try changing this to `syndetroitstation` or something. The only way I see to fix it. If you have no other ideas, it's worth a shot, in my opinion.